pkg/api/client.go

```diff
@@ -120,7 +129,13 @@ func (c *httpClient) prepareAPIRequest(ctx context.Context, request Request) (*h
 		}
 	}
 	if !hasUserAgentHeader {
-		req.Header.Set("User-Agent", config.AgentTypeName+"/"+config.AgentVersion)
+		var deploymentType string
```

Comment:
This can be deploymentType := binary and if IsDocker set it to docker.

Also, can you make these lowercase?